blocks/init/src/Blocks/components/lists/components/lists-editor.js

```diff
@@ -25,6 +25,12 @@ export const ListsEditor = (attributes) => {
 	const listsContent = checkAttr('listsContent', attributes, manifest);
 	const listsOrdered = checkAttr('listsOrdered', attributes, manifest);
 
+	const listsOrderedOptions = getOption('listsOrdered', attributes, manifest).map(option => option.value);
+	const listsDefault = manifest.attributes.listsOrdered.default;
+	if (!listsOrderedOptions.includes(listsOrdered)) {
+		setAttributes({ [getAttrKey('listsOrdered', attributes, manifest)]: listsDefault }); // Resets the attribute to the default value if invalid value set.
```

Comment:
Is there a way to force an invalid value besides manually editing the code view?

If not, maybe this check should be only on PHP side

Reply:
There isn't, but `RichText` is rendered with that HTML tag, so I'm not too sure we should render an attribute we're not sure is safe.